lib/widgets/home_grid_card.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+class HomeGridCard extends StatelessWidget {
+
+  final String title;
+  final IconData icon;
+  final Function function;
+
+  HomeGridCard({
+    @required this.title,
+    @required this.icon,
+    @required this.function
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      child: Container(
+        height: 100,
+        width: 100,
+        decoration: BoxDecoration(
+          color: Colors.white,
+          borderRadius: BorderRadius.circular(30.0),
+        ),
+        child: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Icon(
+                icon,
+                color: Colors.indigo,
+                size: 45.0,
+              ),
+
+              SizedBox(
```

Comment:
```suggestion
              const SizedBox(
```
[Use const constructors whenever possible](https://dev.to/pedromassango/flutter-performance-tips-1-const-constructors-4j41)